.github/templates/main.yml

```diff
@@ -103,7 +103,7 @@ steps:
 #@ end
 
 #@ def buildDocs():
-#@ docsCondition = "${{ contains(github.ref, 'release') }}"
+#@ docsCondition = "${{ contains(github.head.ref, 'release') }}"
```

Comment:
Just for context, in this branch `github.head.ref` = `ac/ci-fix-publish-workflow`. So this is what we want.

Reply:
If I understand correct, the docs will be created whenever a branch contains the word `release`.

Could you rename your branch to something that contains `release` so that we can see this effect?
A.k.a. how would I test this? :)

Reply:
You can just print the content of the var, like I did. That's why I wrote the comment. You can modify the workflow and add:
1. in powershell: `$env:GITHUB_HEAD_REF`
2. in bash: `echo $GITHUB_HEAD_REF`

To any of the steps that use run, or make your own run step. Then, you can see in the workflow output its content.

For an explanation of what all github's env vars contain, you can look [here](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables).

Reply:
![image](https://user-images.githubusercontent.com/4258222/125811569-f5424c3d-a3ff-4459-ab48-af3d2d15ed47.png)
You can see here. The first echo was a test that was not correct, hence a blank line after the new line and then the 3 vars are shown below.

Reply:
Gotcha! Thanks for clarifying.